docs/Projects/TAD/Reporting-Standard/0.1a2.md

```diff
@@ -0,0 +1,407 @@
+# Reporting Standard 0.1a2
```

Comment:
@anneschuth what do you think of keeping the content of the reporting standard in `index.md` file to which we refer to as `latest`? Still with a version number. By doing it this way it can be easier to track changes over time on the reporting standard...and I think it will make it easier to review these changes.

When we create a new version, we can copy the "old" version to a file with the previous version.

Reply:
Agreed, that's better. Or we keep it in latest.md (with a softlink from index.md)